src/components/Footer.js

```diff
@@ -11,6 +11,12 @@ const FooterContainer = styled.div`
   display: flex;
   justify-content: space-between;
   padding: 50px 0 80px 0;
+
+  @media only screen and (max-width: 768px) {
+    justify-content: space-between;
```

Comment:
`justify-content: space-between;` - this is already on the base styles, no need to add it again
also, i dont think this applies to `flex-direction: column` unless you give it a height setting